react/LocationSearch.tsx

```diff
@@ -48,46 +55,69 @@ const LocationSearch: React.FC<LocationSearchProps> = ({ onSelectAddress }) => {
   return (
     <div className={`${styles.locationSearch} w-100`}>
       <Combobox onSelect={handleAddressSelection}>
-        <ComboboxInput
-          as={Input}
-          testId="location-search-input"
-          label={
-            <FormattedMessage id="place-components.label.autocompleteAddress" />
-          }
-          prefix={
-            <div className="c-action-primary flex justify-center items-center">
-              <IconSearch />
-            </div>
-          }
-          suffix={
-            searchTerm.trim().length && (
-              <span
-                data-testid="location-search-clear"
-                role="button"
-                tabIndex={-1}
-                className="pointer c-muted-3 flex justify-center items-center outline-0"
-                onClick={() => setSearchTerm('')}
-                onKeyPress={() => {}}
-              >
-                <IconClear />
-              </span>
+        <div ref={inputWrapperEl}>
```

Comment:
related to the comment above ☝️ 

```suggestion
        <div ref={inputWrapperRef}>
```